app/_src/gateway/kong-enterprise/partials.md

```diff
@@ -15,6 +15,8 @@ Partials address this issue by allowing you to extract shared configurations int
 - **`redis-ce`**: Stores common Redis configurations for plugins available in {{site.ce_product_name}}.
 - **`redis-ee`**: Stores common Redis configurations for plugins available only in {{site.ee_product_name}}.
 
+{:.note}
+> An easy way to determine the required Redis configuration is by examining the fields in use. Redis CE has a shorter and simpler configuration, whereas Redis EE provides options for configuring Redis Sentinel or Redis Cluster for connection.
```

Comment:
```suggestion
An easy way to determine the required Redis configuration is by examining the fields in use. `redis-ce` has a shorter and simpler configuration, whereas `redis-ee` provides options for configuring Redis Sentinel or Redis Cluster for connection.
```
This doesn't need to be a note, it can just be a regular paragraph.